toqito/states/brauer.py

```diff
@@ -69,6 +69,7 @@ def brauer(dim: int, p_val: int) -> np.ndarray:
     matchings = perfect_matchings(2 * p_val)
     num_matchings = matchings.shape[0]
     state = np.zeros((dim ** (2 * p_val), num_matchings))
+    matchings = matchings - 1
```

Comment:
Can you please add a comment here as to why you are using `matchings - 1`? 

It will be helpful if we have to make changes to this function in the future.

Reply:
The perfect matching system Returns  a list of matching which start at an index 1 hence I had to do the -1 but thinking about that, I think its best I fix this at source and make the perfect matching return an array that starts from 0 than one . Does it make sense? 
```
    This is an example of how to generate all perfect matchings of the numbers 1, 2, 3, 4.

    >>> from toqito.perms import perfect_matchings
    >>> perfect_matchings(4)
    array([[1, 2, 3, 4],
           [1, 3, 2, 4],
           [1, 4, 3, 2]])
     ```